biojava-structure/src/main/java/org/biojava/nbio/structure/io/PDBFileParser.java

```diff
@@ -371,13 +371,16 @@ private void pdb_HEADER_Handler(String line) {
 
 			logger.debug("Parsing entry " + pdbId);
 
-
 			PdbId pdbIdToSet;
-			try {
-				pdbIdToSet = new PdbId(pdbCode);
-			} catch (IllegalArgumentException e) {
-				logger.info("Malformed (or null) PDB ID {}. setting PdbId to null", pdbCode);
+			if(pdbCode.isBlank()) {
```

Comment:
It seems that `String.isBlank()` is a java 11 feature that we can't use just yet. At least that's what IntelliJ tells me. Interestingly the tests passed in CI, not sure why.

In any case, I'd like to move to java 11 soon. I think it is long overdue. We'll try to address that separately.